Unaccountable-Magpie/BangazonSite
# Tweak the model and update the migration

### Purpose:

To allow the `City` property of a Product to be null if it's null.`
And to add UserId properties to the Product and Order model

### How it fits in context of the project:

Not all products will have cities.

### Specific Feature affected:

Product.cs (data model) and related migrations, database.

### How to test (Be thorough!):

1. `git checkout el-removeRequiredFromCity`
1. `start Bangazon.sln`
1. in the SQL Server Object Explorer, right click on the BangazonSite DB and click "delete" (check both boxes)
1. In the Package Manager Console run `update-database`
1. check for errors 
1. See if there is a column for 'UserId' on the Order and Product tables
1. See if the Payment Types are still there
